dev-requirements.txt

```diff
@@ -7,6 +7,8 @@ webargs>=0.9.0
 WTForms>=2.0.1
 colander>=1.0
 Flask
+marshmallow
```

Comment:
These are already dependencies of `smore`, so they will be installed after doing 

```python
python setup.py develop
```

so I don't think it's necessary to add these here.